src/test_workflow/integ_test/process_utils.py

```diff
@@ -0,0 +1,40 @@
+class ProcessUtils:
```

Comment:
Missing license header? 
no need to change for this PR, but could we move this utils to a separate folder

Reply:
Thanks for the scrutiny. Added the license header. In terms of the location of the file, I am more inclined to keep it under the current dir since it is for integ test. But we can always iterate over it.